src/api/users/types.d.ts

```diff
@@ -6,3 +6,8 @@ export interface UserResponse {
   photo: string
   provider: LoginProvider
 }
+
+export interface UserResponseCacheEntry {
```

Comment:
In the same vein this could be a a generic CacheEntry<T>…

Reply:
That's true. And I even can move the revision cache into the API-logic and let that use a common cache entry object.

Reply:
Done